### PR TITLE
support gloal image Registry for chart

### PR DESCRIPTION
**Pull request by my-git9**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
support gloal image Registry for chart

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
The test result:
```shell
$ helm upgrade --install kubeedge ./ --dry-run |grep image
        image: docker.io/kubeedge/cloudcore:v1.12.0
        imagePullPolicy: IfNotPresent
$ helm upgrade --set global.imageRegistry=10.0.0.1 --install kubeedge ./ --dry-run |grep image
        image: 10.0.0.1/kubeedge/cloudcore:v1.12.0
        imagePullPolicy: IfNotPresent
```

**Does this PR introduce a user-facing change?**:

```release-note
chart: support gloal image Registry for chart 
```

### CODE REVIEW

File: manifests/charts/cloudcore/values.yaml [hunk 1]
```diff
@@ -3,10 +3,14 @@
...
 appVersion: "1.12.0"
 
+global:
+  imageRegistry: ""
+
 cloudCore:
   replicaCount: 1
   hostNetWork: "true"
   image:
+    registry: docker.io
```
Comment (gy95) [inline manifests/charts/cloudcore/values.yaml:13]: if we specify this two registry both, what's the expected behavior?

Comment (my-git9) [reply manifests/charts/cloudcore/values.yaml:13]: If the images come from different registry (which is rare), we do not need to configure global.
 If they come from the same registry, you can configure global. 
Global first

Comment (zhu733756) [reply manifests/charts/cloudcore/values.yaml:13]: I agree with @gy95, we had better simplify the codes to avoid bringing confusion.

Comment (my-git9) [reply manifests/charts/cloudcore/values.yaml:13]: ok, I will close it

**Pull request closed without merging**